Source/UserManagement/Domain/StaffUser/StaffUser.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Concepts;
+using doLittle.Domain;
+using doLittle.Runtime.Events;
+using Events.StaffUser;
+namespace Domain.StaffUser
+{
+    public class StaffUser : AggregateRoot
+    {
+        protected StaffUser(EventSourceId id) : base(id) {}
+
+        //TODO: I have no idea, not even sure what AggregateRoot means. Someone should look over this.
+        public void AddStaffUser(AddStaffUser command)
```

Comment:
Don't add commands to the Aggregate Root.  The Aggregate Root should expose methods with parameters that are needed.  It's the role of the Command Handler to call the correct method on the aggregate root and to pass the appropriate parameters.

The AddStaffUser command also seems quite "CRUD"y.  A parameter that distinguishes between different types is usually a code smell.  AddAdmin, AddDataConsumer, AddDataCoordinator, etc. are much better commands.  They express the intent without having to interrogate the values of the command.